app/api/events.py

```diff
@@ -168,7 +168,7 @@ def create(self, data, url):
         save_to_db(new_event, "Event saved")
         # set organizer
         role = Role.query.filter_by(name=ORGANIZER).first()
-        uer = UsersEventsRoles(g.user, new_event, role)
+        uer = UsersEventsRoles(user_id=creator_id, event=new_event, role=role)
```

Comment:
Making `creator_id` an optional argument in ```EventDAO.create(self, data, url)``` would do?
[Code](https://github.com/fossasia/open-event-orga-server/blob/development/app/api/events.py#L158)